util/process_wrapper/transitions.bzl

```diff
@@ -0,0 +1,46 @@
+# buildifier: disable=module-docstring
+def _without_process_wrapper_transition_impl(_settings, _attr):
+    """This transition allows rust_* rules to invoke rustc without process_wrapper."""
+    return {"//util/process_wrapper:use_process_wrapper": False}
```

Comment:
Please add `"//command_line_option:compilation_mode": "opt"` into the transition. With that we will ensure that process wrapper is optimized even when the actual project is built in `fastbuild` or `debug`.

Reply:
Done.